src/resources/kubernetesRedis.ts

```diff
@@ -114,6 +117,7 @@ export class KubernetesRedis extends pulumi.ComponentResource {
         architecture: args.architecture || 'replication',
         commonConfiguration,
         auth,
+        metrics: args.metrics || true,
```

Comment:
this will always default to true if metrics is false. you better do this
```suggestion
        metrics: args.metrics ?? true,
```